src/sysen1/netime.128

```diff
@@ -100,45 +100,80 @@ $forma:	save a
 
 hosts:
 
+; Ask the reliable machines first.  They are all on the global
+; Chaosnet and tend to answer quickly.
+
+	[asciz "LOCAL-TIME-SERVER"]
+	[asciz "TIME.CHAOSNET.NET"]
+	[asciz "CHAOS.SWENSON.ORG"]
+	[asciz "CHAOS2.SWENSON.ORG"]
+	[asciz "ROUTER.CHAOSNET.NET"]
+	<(sixbit /ES/)>,,[asciz "ES"]
```

Comment:
Emulated ITS machines are poor time servers (in particular TT ;-)) so comment them out here.

Reply:
Right, I declare TT to be generally unreliable for anything.  It means Temporary Testing, after all.

Reply:
I am going to leave ES, EX, and UP.  At least ES and EX are getting time updates and should have good clocks. I don't know about UP.  However, it is good to have a few more hosts. It doesn't slow things down much.  In fact, when I've been running this during test, all my systems seem to get their times from EX rather than the bridges.  Of course, EX's time is also as good as the bridges.

Reply:
Well, I guess it depends on the emulator and the uptime. Currently, UP and BV have been up a couple of weeks and are over 2 minutes ahead. I recall seeing your hosts also being ahead (the easiest way is "bhostat.py -s time -1").

But then again, UP and BV don't have NETIME configured to work, they pick up the time at boot and start drifting.

Reply:
Well, NETIME has a way of voting to weed out host times that are off. So if you do have time.chaosnet.net and local-host-time responding, in addition to UP with a way-off clock, NETIME won’t use that time.

However, if you can’t get a time from more reliable sources, sometimes it is better to get some (inaccurate) time.  In my case, when you boot my KS instance, pdp10-is cannot provide a clock value to ITS, so without some time from NETIME, it would not get any time set. I think it is better to get a drifted clock than no clock.